spec/konfa_spec.rb

```diff
@@ -91,6 +91,18 @@ def allowed_variables
     end
   end
 
+  context '#get!' do
+    it 'can access a variable that is not nil' do
```

Comment:
Just to cover all the bases, should we add tests checking outcome for things like empty string and the word "nil" and things? To provide face-palm protection for future fixes and refactorings